database/sqlite/sqlite_aclk.h

```diff
@@ -157,7 +154,7 @@ struct aclk_database_cmd {
     struct aclk_completion *completion;
 };
 
-#define ACLK_DATABASE_CMD_Q_MAX_SIZE (2048)
+#define ACLK_DATABASE_CMD_Q_MAX_SIZE (16384)
```

Comment:
do we have any reasoning behind this? Was this increased to support higher number of children?

Reply:
This will be further refined (prevents blocking during chart/dimension creation)